src/main/java/org/mitre/synthea/world/agents/Clinician.java

```diff
@@ -37,17 +37,22 @@
   
   
   public final Random random;
+  public final long identifier;
   public final String uuid;
-  public final long seed;
   public Map<String, Object> attributes;
   private ArrayList<String> servicesProvided;
   private int encounters;
   public long populationSeed;
   
-  public Clinician(long seed) {
-    this.uuid = UUID.randomUUID().toString();
-    this.seed = seed; // keep track of seed so it can be exported later
-    random = new Random(seed);
+  public Clinician(long clinicianSeed, Random clinicianRand, Boolean preserveClinicianUUID, long identifier) {
```

Comment:
Hmm... it seems like `Boolean preserveClinicianUUID` is an unnecessary argument... doesn't it?

I'm thinking we should just use the ` this.uuid =  new UUID(clinicianSeed, identifier).toString();` assignment in all cases, because a `clinicianSeed` will always be passed in (by default it matches the runtime seed).

If the user provides a seed at runtime with `-cs N` then don't we always want a reproducible UUID based on that seed, regardless of whether or not the `exporter.practitioner.preserve_uuid` property was set?

Am I missing something?

Reply:
Good point. It's not necessary if we're okay with the way UUIDs are generated. I removed it from the code.